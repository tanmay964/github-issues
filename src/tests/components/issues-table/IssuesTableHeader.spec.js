import React from 'react';
import renderer from 'react-test-renderer';
import IssuesTableHeader from '../../../components/issues-table/IssuesTableHeader';

it('renders correctly', () => {
  const tree = renderer
    .create(<IssuesTableHeader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
