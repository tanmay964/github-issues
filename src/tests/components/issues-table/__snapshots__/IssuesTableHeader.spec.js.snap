// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-htpNat dcecVv"
>
  <div
    className="sc-bxivhb cRqBkp"
  >
    <a
      className="sc-ifAKCX hARpCS"
      href="https://github.com/facebook/react/issues?q=is%3Aopen+is%3Aissue"
      selected={true}
    >
      <svg
        aria-hidden="true"
        className="sc-bdVaJa emxeyh"
        color="#fff"
        height="16"
        selected={true}
        width="14"
      >
        <path
          d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"
        />
      </svg>
      253 Open
    </a>
    <a
      className="sc-ifAKCX iMVwXm"
      href="https://github.com/facebook/react/issues?q=is%3Aopen+is%3Aissue"
      style={
        Object {
          "marginLeft": "10px",
        }
      }
    >
      <svg
        aria-hidden="true"
        className="sc-bdVaJa hUOxzg"
        color="#586069"
        height="16"
        selected={true}
        width="12"
      >
        <path
          d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"
          fillRule="evenodd"
        />
      </svg>
      6378 closed
    </a>
  </div>
  <div
    className="sc-bZQynM fTxYOY"
  >
    <span
      className="sc-EHOje kvbWuD"
    >
      Auther
       
      <span
        className="sc-bwzfXH bdTERA"
      />
       
    </span>
    <span
      className="sc-EHOje kvbWuD"
    >
      Label
       
      <span
        className="sc-bwzfXH bdTERA"
      />
       
    </span>
    <span
      className="sc-EHOje kvbWuD"
    >
      Projects
       
      <span
        className="sc-bwzfXH bdTERA"
      />
       
    </span>
    <span
      className="sc-EHOje kvbWuD"
    >
      Milestones
       
      <span
        className="sc-bwzfXH bdTERA"
      />
       
    </span>
    <span
      className="sc-EHOje kvbWuD"
    >
      Assignee
       
      <span
        className="sc-bwzfXH bdTERA"
      />
       
    </span>
    <span
      className="sc-EHOje kvbWuD"
    >
      Sort
       
      <span
        className="sc-bwzfXH bdTERA"
      />
       
    </span>
  </div>
</div>
`;
